Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,25 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
```

Comment:
All comments that I wrote can be removed. It was just for you, to have a hint where to write what. Not needed after the fact you wrote something.

Reply:
Comments have been removed as suggested.

Reply:
Comments have been removed as suggested.